src/dashboard/Data/Config/ConfigDialog.react.js

```diff
@@ -228,6 +233,30 @@ export default class ConfigDialog extends React.Component {
               description='Use this to configure your app. You can change it at any time.' />
           }
           input={EDITORS[this.state.type](this.state.value, (value) => { this.setState({ value }) })} />
+        
+        {
+          /*
+            Add `Requires master key` field if parse-server version >= 3.8.0,
+            that is the minimum version that supports this feature.
+          */
+          semver.valid(this.props.parseServerVersion) && semver.gte(this.props.parseServerVersion, '3.8.0') 
```

Comment:
We use to check the [server info](https://github.com/parse-community/parse-server/blob/master/src/Routers/FeaturesRouter.js) to decide if a feature is available or not at Parse Dashboard. What do you think?

Reply:
I took this approach from here:
https://github.com/parse-community/parse-dashboard/blob/169e13b2328fc5adff2afffa2f6e855eb7cc638e/src/dashboard/Data/Browser/AddColumnDialog.react.js#L217-L218

Reply:
And as I understand it, `app.serverInfo` is set by calling the `/serverInfo` endpoint when the dashboard is launched:
https://github.com/parse-community/parse-dashboard/blob/169e13b2328fc5adff2afffa2f6e855eb7cc638e/src/dashboard/Dashboard.js#L129-L131